app/assets/javascripts/validation.coffee

```diff
@@ -605,18 +612,20 @@ $(document).ready ->
 
   # Validate the whole page on page load
   if $('.metadata-form, .umm-form').length > 0
-    # "visit" each field with a value on page load
-    $('.validate').not(':disabled').filter ->
-      return switch this.type
-        when 'radio'
-          # don't want to save fields that aren't translated into metadata
-          this.name? and this.checked
-        else
-          this.value
-    .each (index, element) ->
-      visitField($(element).attr('id'))
+    # Do not display validation errors on page load if model errors are showing
+    if $('.errors').length == 0
```

Comment:
I would prefer to add and use a more specific class for this check, maybe `.model-errors`

Reply:
Done